source/application/DbgHelp/DbgHelpStackWalkCallback.cpp

```diff
@@ -188,7 +331,8 @@ void DbgHelpStackWalkCallback::EnumerateParameters()
         m_unsure++;
         for (DWORD i = 0; i < 4; i++)
         {
-            m_param_info.emplace_back(ParamInfo{ DbgHelpArgType::Int, L"int", { static_cast<WCHAR>(L'a' + i) }, nullptr, nullptr, 4, i * 4 });
+            m_param_info.emplace_back(ParamInfo{ DbgHelpBasicType(DbgHelpBasicType::BasicType::Int32),

```

Comment:
It would be preferable to name the variables something, even if we don't know their names. Like IDA does. That was what I was trying to achieve with `L'a' + i` (as that would name them a, b, c, d, and so forth).